astropy/units/core.py

```diff
@@ -346,7 +346,8 @@ def add_enabled_units(units):
 
     Parameters
     ----------
-    units : list of sequences, dicts, or modules containing units, or units
+    units : list of sequence, dict, or module
+        The list of X must contain units or units.
```

Comment:
Don't know why I didn't see this, but the additional sentence does not seem to help. Just remove?

Reply:
✅